content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md

```diff
@@ -16,7 +16,7 @@ The additional APIs can either be ready-made solutions such as a
 [metrics server](https://github.com/kubernetes-sigs/metrics-server), or APIs that you develop yourself.
 
 The aggregation layer is different from
-[Custom Resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/),
+[Custom Resources Definitions](/docs/concepts/extend-kubernetes/api-extension/custom-resources/),
```

Comment:
Did you mean:
```suggestion
[Custom Resource Definitions](/docs/concepts/extend-kubernetes/api-extension/custom-resources/),
```

Reply:
Yes @sftim, thanks for the correction. I will update the PR.